src/dataflow/src/source/mod.rs

```diff
@@ -556,8 +564,8 @@ impl SourceMetrics {
             )
             .unwrap();
             static ref CAPABILITY: UIntGaugeVec = register_uint_gauge_vec!(
-                "mz_kafka_capability",
-                "The current capability for this dataflow. This corresponds to min(mz_kafka_partition_closed_ts)",
+                "mz_capability",
+                "The current capability for this dataflow. This corresponds to min(mz_partition_closed_ts)",
                 &["topic", "source_id", "worker_id"]
             )
             .unwrap();
```

Comment:
(I can't comment outside the range of the change context)

Similarly, below on lines 590 to 609, it seems like either we should have different kinds of metrics depending on the source, or at least an extra `"source_type"` label that can be e.g. `"file"`, `"kafka"`, `"kinesis"`